packages/@aws-cdk/aws-rds/test/integ.cluster.ts

```diff
@@ -1,23 +1,19 @@
 import * as ec2 from '@aws-cdk/aws-ec2';
 import * as kms from '@aws-cdk/aws-kms';
+import * as s3 from '@aws-cdk/aws-s3';
 import * as cdk from '@aws-cdk/core';
 import { DatabaseCluster, DatabaseClusterEngine } from '../lib';
-import { ClusterParameterGroup } from '../lib/parameter-group';
 
 const app = new cdk.App();
 const stack = new cdk.Stack(app, 'aws-cdk-rds-integ');
 
 const vpc = new ec2.Vpc(stack, 'VPC', { maxAzs: 2 });
 
-const params = new ClusterParameterGroup(stack, 'Params', {
-  family: 'aurora5.6',
-  description: 'A nice parameter group',
-  parameters: {
-    character_set_database: 'utf8mb4'
-  }
-});
-
 const kmsKey = new kms.Key(stack, 'DbSecurity');
+
+const importBucket = new s3.Bucket(stack, 'ImportBucket');
+const exportBucket = new s3.Bucket(stack, 'ExportBucket');
+
```

Comment:
Avoid modifying existing integration tests. Add new ones instead.

Optionally, if there is a simple way to verify that s3 import and export works, after deploying the stack, document that too. See [here](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests) for some examples.

Reply:
Done.

The verification steps are a bit too comprehensive I think. First need to get access through a tunnel or similar, then create a table, insert data, export into bucket, import it back into database etc.